05/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> HTML i CSS: Responsywność</title>
+
+    <!-- Styles Links -->
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/units.css">
+    <link rel="stylesheet" media="(max-width: 800px)" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 801px)" href="./styles/desktop.css">
+</head>
+<body>
+
+<!--     
+=================
+Nav Section
+=================
+-->
+
+<nav class="nav">
+    <section class="nav__section">
+        <div class="container flex">  
+            <div class="content nav__logo"></div>
+        
+            <div class="content nav__menu">
+                <!-- Hamburger Button -->
+                <input id="menu__toggle" type="checkbox" />
+                <label class="menu__btn" for="menu__toggle">
```

Comment:
Nie widzę nigdzie, Blocku o nazwie `menu` w BEM hierarchia musi być zachowania - jeśli występuje ELEMENT to musi być też BLOCK. Czyli np. do div z 28 linii dodałbym klasę menu -> `<div class="content nav__menu menu">`

Tutaj znajdziesz więcej informacji: https://devmentor.pl/b/metodologia-bem-w-css-i-sass

Reply:
poprawione